src/server/index.tsx

```diff
@@ -18,7 +19,7 @@ const [hostname, port] = process.env["LEMMY_UI_HOST"]
 
 server.use(express.json());
 server.use(express.urlencoded({ extended: false }));
-server.use("/static", express.static(path.resolve("./dist")));
+server.use(`/static-${COMMIT_HASH}`, express.static(path.resolve("./dist")));
```

Comment:
I believe several instances have already set up their configurations to work based on the assumption that `/static/` is a path to cacheable files. Renaming this directory will most likely break those existing configurations. 

What do you think about adding a new subdirectory instead? So instead of `/static-af29123/` we could have `/static/af29123/`.